llmfoundry/callbacks/hf_checkpointer.py

```diff
@@ -201,9 +204,18 @@ def save_model_patch(*args: Any, **kwargs: Any):
     if mlflow_logger.model_registry_uri is not None:
         mlflow.set_registry_uri(mlflow_logger.model_registry_uri)
 
+    if is_peft:
+        transformers_in_memory_model = pipeline(
```

Comment:
won't this not be a peft model?

Reply:
shouldn't you just pass the in memory model into this function directly for the peft case?

Reply:
For some reason mlflow requires either a Pipeline or a path to the model.